cellprofiler/gui/pipelinecontroller.py

```diff
@@ -2112,36 +2112,30 @@ def add_paths_to_pathlist(self, filenames):
             style=wx.PD_APP_MODAL | wx.PD_CAN_ABORT,
         ) as dlg:
             assert isinstance(dlg, wx.ProgressDialog)
+
+            # Set dialog width to at least 480
+            min_width = 480
             h, w = dlg.GetSize()
-            if w < 480:
-                dlg.SetSize((max(w, 480), h))
+            if w < min_width:
+                dlg.SetSize(min_width, h)
+
             queue = six.moves.queue.Queue()
             interrupt = [False]
             message = ["Initializing"]
 
             def fn(filenames=filenames, interrupt=None, message=None, queue=queue):
-                if message is None:
-                    message = message
-                if interrupt is None:
```

Comment:
Are we asserting that interrupt will now exist? It seems possible for it to be `None` in which case `interrupt[0]` will fail.

Reply:
hm... i might have made a mistake here; i'll need to run some tests and figure out the details regarding threading and sharing variables.